Parse.py

```diff
@@ -0,0 +1,35 @@
+# NOTE: To simplify the coding, just feed in consecutive batch IDs with
+# num_process in mind (e.g. if num_process=3, array=0 would process the first
+# 3 months in the dates array)
+
+# NOTE: The very last batch should be run individually, so as not to mess up
+# the aggregation across all of the months
+
+# BUG: Because of a hacky solution within lang_filtering(), the language filtering
+# would only work properly for fully consecutive set of months within self.dates
+# TODO: make it more general
+
+import subprocess
+import time
+import sys
+from pycorenlp import StanfordCoreNLP
+from reddit_parser import Parser
+import argparse
+import numpy
+from defaults import *
+
+if __name__ == "__main__":
+    argparser = argparse.ArgumentParser()
+    argparser.add_argument('--array', type = int)
+    argparser.add_argument('--machine', type = str)
+    args = argparser.parse_args()
```

Comment:
In our documentation, we need to describe how running on the cluster and locally would work differently. For instance, this bit here would be needed for running on CCV to work, but would give an error if run locally

Reply:
Great, made a task for this.